docs/modules/clients/pages/clc.adoc

```diff
@@ -0,0 +1,350 @@
+
+`hzc -c /some/custom/config.yaml`
+
+You can always override some fields of the configuration file with the following top level flags:
+
+- *--address/-a*: Addresses of the instances in the cluster (default is localhost:5701).
+- *--cloud-token*:  Hazelcast Cloud token
+- *--cluster-name*: name of the cluster to connect (default is dev)
+
+=== Configuring SSL
+
+You can use the following configuration file to enable SSL support:
+
+[source,yaml]
+----
+ssl:
+  servername: "HOSTNAME-FOR-SERVER"
+  # or: insecureskipverify: true
+hazelcast:
+  cluster:
+    security:
+      credentials:
+        username: "OPTIONAL USERNAME"
+        password: "OPTIONAL PASSWORD"
+    name: "CLUSTER-NAME"
+    network:
+    addresses:
+    - "localhost:5701"
+----
+
+Mutual authentication is also supported:
+[source,yaml]
+----
+ssl:
+    servername: "HOSTNAME-FOR-SERVER"
+    # insecureskipverify: true
+    capath: "/tmp/ca.pem"
+    certpath: "/tmp/cert.pem"
+    keypath: "/tmp/key.pem"
+    keypassword: "PASSWORD FOR THE KEY"
+hazelcast:
+  cluster:
+    security:
+      credentials:
+        username: "OPTIONAL USERNAME"
+        password: "OPTIONAL PASSWORD"
+    name: "CLUSTER-NAME"
+    network:
+      addresses:
+        - "localhost:5701"
+----
+
+Cloud SSL configuration:
+
+[source,yaml]
+----
+ssl:
+    capath: "/tmp/ca.pem"
+    certpath: "/tmp/cert.pem"
+    keypath: "/tmp/key.pem"
+    keypassword: "PASSWORD FOR THE KEY"
+hazelcast:
+  cluster:
+    name: "CLUSTER NAME"
+    cloud:
+      token: "HAZELCAST CLOUD TOKEN"
+      enabled: true
+----
+
+=== Modes
+
+*hzc* packs 3 usage modes to simplify different use cases.
+
+*Default (Non-interactive)*
+
+You can execute commands similar to any other command line tool. It can also be useful for scripting.
+[source,bash,subs="attributes+"]
+----
+hzc map --cluster-name cluster1 --name myMap put --key myKey --value myValue
+----
+
+*Interactive*
+
+This starts an interactive prompt, in which you can interactively navigate commands and explore possible subcommands and flags. It is also useful for manual tasks.
+
+image:ROOT:hzc-interactive-screenshot.png[HZC interactive]
+
+[source,bash,subs="attributes+"]
+----
+hzc
+----
+
+On this mode, you can also leverage the "use" command to avoid re-typing the distributed object name on each command. Note that map name "m1" is referenced only once.
+----
+hzc
+Connecting to the cluster ...
+hzc localhost:5701@dev> map use m1
+hzc localhost:5701@dev&m:m1> map put -k k1 -v v1
+hzc localhost:5701@dev&m:m1> map get -k k1
+v1
+hzc localhost:5701@dev&m:m1>
+----
+
+*SQL Browser*
+
+All the modes support executing SQL statements. In addition to that, this mode makes it easier to navigate results with an interactive browser.
+
+image:ROOT:hzc-sql-browser-screenshot.png[HZC SQL Browser]]
+
+To start the browser simply type
+
+[source,bash,subs="attributes+"]
+----
+hzc sql
+----
+
+== Commands
+
+=== Distributed Data Structures
```

Comment:
```suggestion
```

I don't think we need this. It doesn't add any value and we don't introduce the section with any additional details.

Reply:
Makes sense thanks https://github.com/hazelcast/hz-docs/pull/462/commits/b3e94f20a0864ac722d6b633d8dc5a57e9758e7b